conf/article_validator.yaml

```diff
@@ -0,0 +1,721 @@
+          - 312 #mottled red
+          - 313 #mottled dark
+          - 314 #mottled bordeaux
+          - 706 #copper
+          - 315 #metallic red
+          - 401 #rose
+          - 402 #pink
+          - 411 #mottled rose
+          - 412 #mottled pink
+          - 915 #neon pink
+          - 916 #nude
+          - 605 #petrol
+          - 615 #mottled teal
+          - 204 #apricot
+          - 611 #mottled light
+          - 612 #mottled green
+          - 613 #mottled dark
+          - 913 #neon green
+          - 616 #metallic green
+          - 101 #light grey
+          - 102 #grey
+          - 103 #dark grey
+          - 105 #stone
+          - 111 #mottled light
+          - 112 #mottled grey
+          - 113 #mottled dark
+          - 956 #grey denim
+          - 114 #metallic grey
+          - 115 #gunmetal
+          - 206 #gold
+          - 217 #rose gold
+          - 201 #light yellow
+          - 202 #yellow
+          - 203 #dark yellow
+          - 211 #mottled light
+          - 212 #mottled yellow
+          - 213 #mottled dark
+      supplierColor:
+        description: |
+          This is the color that was provided by the articles manufacturer or
+          supplier.
+        type: string
+        minLength: 3
+        maxLength: 50
+      supplierSKU:
+        description: |
+          Supplier SKU of ArticleConfig valid for specific color scheme of
+          Article described by ArticleConfig
+        type: string
+      genders:
+        description: |
+          Genders for which to
+          Product is sold. Multiple Genders
+          are possible
+        type: array
+        items:
+          type: string
+          enum:
+            - female
+            - male
+        maxItems: 2
+      season:
+        description: |
+          Available
+          Seasons
+          are: HW[XX] -
+          "Herbst-Winter"
+          (Fall/Winter)
+          where as XX
+          represents the year as two
+          digit
+          number, FS[XX] -
+          "Frühjahr-Sommer"
+          (Spring/Summer) where as XX
+          represents the year as two
+          digit
+          number, NOS (Never
+          Out
+          of Stock) - can always be
+          sold
+        type: array
+        items:
+          type: string
+          minLength: 3
+          maxLength: 4
+          pattern: /((HW|FS)[0-9]{2,2}|NOS|HWBASIS|FSBASIS|GANZJAHR|AUTUMN_WINTER_BASICS|SPRING_SUMMER_BASICS|YEAR_ROUND_BASICS)/
+      images:
+        type: array
+        items:
+          type: object
+          description: A single Article Image with its sort order index and url
+          properties:
+            sortOrder:
+              type: number
+              maximum: 7
+              minimum: 1
+            url:
+              type: string
+              format: url
+        maxItems: 7
+
+      # --- Optional properties ------------------------------------------------------------------->
+      mainMaterial:
```

Comment:
Consider remodelling config-material-color relationships to come up with something DRY and flexible so that we can transform it to what is expected by Zeos